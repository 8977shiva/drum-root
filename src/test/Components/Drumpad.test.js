import React from "react";
import { shallow } from "enzyme";
import Drumpad from "../../Components/DrumPad";
import { mockLayout1 } from "../../mockData/mockLayout";

describe("Drumpad component", () => {
  describe("component rendering", () => {
    test("renders properly", () => {
      const wrapper = shallow(<Drumpad layout={mockLayout1} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
