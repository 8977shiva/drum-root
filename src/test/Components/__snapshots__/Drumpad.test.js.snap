// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drumpad component component rendering renders properly 1`] = `
<styled.table>
  <tbody>
    <tr
      key="hiHat"
      onClick={[Function]}
    >
      <td
        key="0"
        onClick={[Function]}
      >
        <span
          aria-label="hiHat"
          role="img"
        >
          🇹🇼
        </span>
      </td>
      <td
        key="1"
        onClick={[Function]}
      >
        <span
          aria-label="hiHat"
          role="img"
        >
          🇹🇼
        </span>
      </td>
      <td
        key="2"
        onClick={[Function]}
      >
        <span
          aria-label="hiHat"
          role="img"
        >
          🇹🇼
        </span>
      </td>
      <td
        key="3"
        onClick={[Function]}
      >
        <span
          aria-label="hiHat"
          role="img"
        >
          🇹🇼
        </span>
      </td>
    </tr>
    <tr
      key="bass"
      onClick={[Function]}
    >
      <td
        key="0"
        onClick={[Function]}
      >
        <span
          aria-label="bass"
          role="img"
        >
          🛢️
        </span>
      </td>
      <td
        key="1"
        onClick={[Function]}
      />
      <td
        key="2"
        onClick={[Function]}
      />
      <td
        key="3"
        onClick={[Function]}
      />
    </tr>
    <tr
      key="snare"
      onClick={[Function]}
    >
      <td
        key="0"
        onClick={[Function]}
      />
      <td
        key="1"
        onClick={[Function]}
      />
      <td
        key="2"
        onClick={[Function]}
      >
        <span
          aria-label="snare"
          role="img"
        >
          🥁
        </span>
      </td>
      <td
        key="3"
        onClick={[Function]}
      />
    </tr>
  </tbody>
</styled.table>
`;
